client/src/modules/stock/movements/modals/search.modal.js

```diff
@@ -1,39 +1,22 @@
 angular.module('bhima.controllers')
 .controller('SearchMovementsModalController', SearchMovementsModalController);
 
-// dependencies injections
 SearchMovementsModalController.$inject = [
-  'DepotService', 'InventoryService', 'NotifyService', '$uibModalInstance', 
-  'SearchFilterFormatService', 'FluxService', '$translate'
+  'data', 'NotifyService', '$uibModalInstance', 'FluxService', 
+  '$translate', 'PeriodService', 'Store', 'util'
 ];
 
-function SearchMovementsModalController(Depots, Inventory, Notify, Instance, SearchFilterFormat, Flux, $translate) {
+function SearchMovementsModalController(data, Notify, Instance, Flux, $translate, Periods, Store, util) {
```

Comment:
Instead of calling this parameter "data", it might make more sense to call this `filters`.